bin/scalafix

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
-VERSION="0.6.0-M12-patch1"
+# NOTE: Published from https://github.com/xeno-by/scalafix/pull/4
+VERSION="0.6.0-M12-patch1-1-9a805cea-SNAPSHOT"
```

Comment:
This brings @maxov's improvements to MacroExpansionTree (https://github.com/scalameta/scalameta/pull/1722) and my improvements to m.Position (https://github.com/scalameta/scalameta/pull/1723).